src/pages/messages/components/TicketDateIntervalFilter.tsx

```diff
@@ -53,7 +46,9 @@ export const TicketDateIntervalFilter = () => {
         maxDate={selectedToDate ? new Date(selectedToDate) : maxDate}
         onChange={(date, context) => {
           if (context.validationError !== 'invalidDate') {
-            onChangeFromDate(date);
+            if (isEmptyOrStartsWithFourDigitYear(date ? formatDateStringToISO(date) : '')) {
+              updateSearchParams(date ? formatDateStringToISO(date) : '', selectedToDate);
+            }
```

Comment:
samme operasjon gjøres 2 ganger her. Mulig å slå sammen `formatDateStringToISO(date)` til å bare kalles en gang? Ser ut som vi enten skal nullstille eller sette en verdi, så dette er kanskje det samme (ikke testet, og kan sikkert gjøres enda mer elegant også)

```suggestion
            if (!date) {
              updateSearchParams('', selectedToDate);
            } else {
              const isoDate = formatDateStringToISO(date);
              if (startsWithFourDigitYear(isoDate)) {
                updateSearchParams(isoDate);
              }
            }
```

Reply:
When in Rome 👍